srcpkgs/PrismLauncher/template

```diff
@@ -41,9 +41,3 @@ pre_configure() {
 post_install() {
 	vdoc ${FILESDIR}/README.voidlinux
 }
-
-PolyMC_package() {
```

Comment:
No, do not remove this, it's there for a reason: to let people migrate from poly to prism smoothly

Reply:
But it builds Polymc alongside Prism, it causes errors during building process… With all the changes brought to Prism and the drama (and the eventual death) of Polymc, it’s fair to assume that people are aware of the change. Otherwise, they’d still be on multimc.

but fair enough.

Reply:
I’ll rework the template later-on, this evening

Reply:
What errors? The polymc package is just an empty package

Reply:
On my void container, it went okay, but the github build verifications went crazy for some reasons… idk if it was an error or if something is wrong with this commit. Tho, when i removed polymc function, it worked with github verifications.

Reply:
the thing that failed was [a test during the check phase](https://github.com/void-linux/void-packages/actions/runs/5297883808/jobs/9589924723#step:7:876), it had nothing to do with the polymc subpackage. if you want to run the same thing locally, run xbps-src with `-Q` to run checks for packages